ketcher-autotests/tests/User-Interface/Select-Tools/select-bonds.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { test } from '@playwright/test';
+import {
+  takeEditorScreenshot,
+  clickInTheMiddleOfTheScreen,
+  LeftPanelButton,
+  selectLeftPanelButton,
+  selectButtonById,
+  BondTypeId,
+} from '@utils';
+import { getBondByIndex } from '@utils/canvas/bonds';
+
+test.describe('Bonds selection ', () => {
```

Comment:
I think we should add a test where atoms are selected as well as bonds, if I understood the fix correctly

Reply:
Agree with Yahor.